crates/handlers/src/oauth2/mod.rs

```diff
@@ -59,7 +59,11 @@ pub fn filter(
         .unify()
         .or(introspection)
         .unify()
-        .with(cors().allow_methods([Method::POST, Method::GET]));
+        .with(
+            cors()
+                .allow_methods([Method::POST, Method::GET])
+                .allow_headers(["authorization"]),
```

Comment:
There are constants in hyper: <https://docs.rs/hyper/latest/hyper/header/constant.AUTHORIZATION.html>, so
```rust
use hyper::header::AUTHORIZATION;
```

and

```suggestion
                .allow_header(AUTHORIZATION),
```